src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -156,6 +156,8 @@ Consumer<Message<UUID>> consumeCaseImportStart() {
             UUID caseUuid = message.getPayload();
             String variantId = message.getHeaders().get(NotificationService.HEADER_VARIANT_ID, String.class);
             String reportUuidStr = message.getHeaders().get(NotificationService.HEADER_REPORT_UUID, String.class);
+            String caseFormatStr = message.getHeaders().get(NotificationService.HEADER_CASE_FORMAT, String.class);
+            String caseNameStr = message.getHeaders().get(NotificationService.HEADER_CASE_NAME, String.class);
```

Comment:
Maybe not a big deal, but I don't see tests on caseName ?

Reply:
very good remark , in fact case name in not message. i fixed the code now